opencti-platform/opencti-front/src/private/components/settings/users/UserHistory.js

```diff
@@ -31,29 +21,29 @@ class UserHistory extends Component {
     const { t, userId } = this.props;
     const { entitySearchTerm } = this.state;
     return (
-      <div style={{ marginTop: 50 }}>
+      <div>
         <Typography
           variant="h4"
           gutterBottom={true}
-          style={{ float: 'left', marginTop: 12 }}
+          style={{ float: 'left', margin: '16px 0 0 0' }}
```

Comment:
you can use marginTop directly

Reply:
no I can't, if I do this, there is a gap (see screenshot below)

![image](https://user-images.githubusercontent.com/75783086/230041606-e8367514-372f-47b0-85dd-c2b819ebe65b.png)